src/dotnet/APIView/APIViewWeb/Repositories/ReviewManager.cs

```diff
@@ -586,20 +586,21 @@ public async Task UpdateReviewBackground()
             string codeFileName,
             MemoryStream originalFileStream,
             string baselineCodeFileName = "",
-            MemoryStream baselineStream = null
+            MemoryStream baselineStream = null,
+            string project = ""
```

Comment:
Do we want this to be empty here?

Reply:
Thanks. this should be defaulted to public. I will check and fix it.